src/file-manager/fm-list-view.c

```diff
@@ -2891,9 +2891,10 @@ fm_list_view_start_renaming_file (FMDirectoryView *view,
                                   NULL,
                                   list_view->details->file_name_column,
                                   TRUE, 0.0, 0.0);
-    gtk_tree_view_set_cursor (list_view->details->tree_view,
+    gtk_tree_view_set_cursor_on_cell (list_view->details->tree_view,
                               path,
                               list_view->details->file_name_column,
+                              GTK_CELL_RENDERER (list_view->details->file_name_cell),
                               TRUE);
```

Comment:
I guess the alignment should be kept.  BTW upstream commit had it: https://git.gnome.org/browse/nautilus/commit/?id=5b09cedf434b53a975074b346d511335fc735cba